src/target/lmi.c

```diff
@@ -84,6 +84,7 @@ bool lmi_probe(target *t)
 		t->driver = lmi_driver_str;
 		target_add_ram(t, 0x20000000, 0x10000);
 		lmi_add_flash(t, 0x80000);
+		t->target_options |= CORTEXM_TOPT_INHIBIT_SRST;
```

Comment:
I think this line needs a comment saying why we inhibit this. Something like "The Tiva platform is resetting the debug subsystem together with the system reset preventing us from using SRST" or similar.